Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md

```diff
@@ -10,19 +10,20 @@
 
 ## Issue
 
-Service Fabric clusters using Certificate Common Name and Issuer thumbprint 1fb86b1168ec743154062e8c9cc5b171a4b7ccb4 may become unresponsive or have state 'Upgrade Service Unreachable'.  
+For certificates issued by Digicert intermediate 1fb86b1168ec743154062e8c9cc5b171a4b7ccb4, when chains are built on Windows, the chain may be built with the new Digicert intermediate, 626d44e704d1ceabe3bf0d53397464ac8080142c, which shares a public key with 1fb8. Impacted clusters may become unresponsive and may show state as 'Upgrade Service Unreachable' as some SF components are unable to authenticate to each other.
```

Comment:
This may be trying to say too much at once. Please consider "Service Fabric clusters secured with DigiCert certificates declared by Common Name are at risk of failing validation, if the certificate declaration pins the issuer thumbprints."